drivers/display/Kconfig.mcux_elcdif

```diff
@@ -53,6 +53,18 @@ config MCUX_ELCDIF_PXP
 	  display_write is called with a framebuffer equal in size to the
 	  display.
 
+config MCUX_ELCDIF_LP
+	bool "ELCDIF low power"
+	help
+	  This option, when enabled, will enable CurFrameDoneInterrupt at the display
```

Comment:
is `CurFrameDoneInterrupt` what it is called in the reference manual? the description should match the RM not the SDK

Reply:
Effectively, I've changed the description to match with the RM. Thanks.